QuizApp/Presentation/Common/Interfaces/BaseViewController.swift

```diff
@@ -1,8 +1,11 @@
 import Foundation
 import UIKit
+import Combine
 
 class BaseViewController: UIViewController {
 
+    var cancellables: [AnyCancellable] = []
```

Comment:
Please do not add cancellables in BaseViewController. Instead, init it in each VC. Also, use Set<AnyC>